### PR TITLE
Update rq_gevent_worker.py

### PR DESCRIPTION
**Pull request by ggold6**
change from rq0.46 to 0.5.0: Status to JobStatus in job.py

### CODE REVIEW

File: rq_gevent_worker.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import gevent
 import gevent.pool
 from rq import Worker
-from rq.job import Status
+from rq.job import JobStatus
```
Comment (zhangliyong) [inline rq_gevent_worker.py]: We need to compatible with old rq, check rq version to decide importing Status or JobStatus.

**Comment by ggold6** [comment]
added rq<=0.46 import if JobStatus from rq>=0.5.0 import fails.

File: rq_gevent_worker.py [hunk 2]
```diff
@@ -10,7 +10,10 @@
 import gevent
 import gevent.pool
 from rq import Worker
-from rq.job import Status
+try: # for rq >= 0.5.0
+    from rq.job import JobStatus
+except: # for rq <= 0.46
```
Comment (zhangliyong) [inline rq_gevent_worker.py]: prefer: `except ImportError:`
Thanks.

**Comment by ggold6** [comment]
Yep, ImportError should be there. Also fixed comment to proper version # (0.4.6, not 0.46)

**Comment by zhangliyong** [comment]
Thanks!

**Pull request merged**